### PR TITLE
T3 disclaimer

### PR DESCRIPTION
**Pull request by rupertbates**
## What are you doing in this PR?

This PR adds the new approved disclaimer text and T&Cs link to the Tier Three checkout

[**Trello Card**](https://trello.com/c/pUvH8d1p/920-tier-legal-disclaimer-and-link-to-new-tcs-copy-to-be-added-to-new-tier-3-checkout)
[**Doc containing agreed disclaimer text**](https://docs.google.com/document/d/1WmhVI_44gdKj7Eztzh5xmzSxVh8d7y22/edit)
[**Overall T3 progress doc**](https://docs.google.com/document/d/1exq06PZVjPwJ88o2vV8yeU9Fd-C0ey6DBGQYxGkGOB0/edit)
[**New T3 T&Cs page on dotcom**](https://docs.google.com/document/d/1exq06PZVjPwJ88o2vV8yeU9Fd-C0ey6DBGQYxGkGOB0/edit)
## Screenshots
Screenshot 2024-07-23 at 17 39 10

### CODE REVIEW

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx [hunk 1]
```diff
@@ -27,57 +28,38 @@ const termsLink = (linkText: string, url: string) => (
 
 export default function ThreeTierTerms(props: {
 	paymentFrequency: 'month' | 'year';
+	manageLink: EmotionJSX.Element;
```
Comment (andrewHEguardian) [inline support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:31]: does this need to be a prop? It seems to be unchanging in both places its passed

**Review by andrewHEguardian — APPROVED** [review]
Looks good 👍 🧐  tried to read it carefully!

File: support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx (hunk 1 above)
Comment (andrewHEguardian) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx:31]: Ah fair enough, separate file would be the easiest way but doesn't seem worthwhile

Comment (rupertbates) [reply support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx]: I've extracted it to a separate file, I think that is the best approach

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**